libs/utils/src/Card/memory_card.ts

```diff
@@ -103,6 +103,25 @@ export class MemoryCard implements Card {
     this.longValue = Uint8Array.from(value);
   }
 
+  /**
+   * Writes new short and long values to the card.
+   */
+  async writeShortAndLongValues({
+    shortValue,
+    longValue,
+  }: ShortAndLongValues): Promise<void> {
+    await this.writeShortValue(shortValue);
+    await this.writeLongUint8Array(new TextEncoder().encode(longValue));
+  }
+
+  /* istanbul ignore next: A no-op but necessary to satisfy the Card interface */
+  /**
+   * Overrides card write protection.
+   */
+  async overrideWriteProtection(): Promise<void> {
+    return Promise.resolve();
+  }
```

Comment:
I _could_ implement a write-protection mechanism for `MemoryCard` and have it mimic the (fairly complex) circumstances under which our web service cards automatically enable and disable write protection, but that didn't feel worth the trouble, especially when we're considering revisiting the write-protection mechanism anyway